### ISSUE TITLE
Unity sucks ass

### ISSUE THREAD

**Issue by Aiz0** [Aiz0/dotless#7]
Unity is bad.

Unity Hub
- Can't open existing projects
- Still thinks unity editor is in default directory even though i changed it in settings
- Creates a `~/.local/UnityHub` directory

Unity Editor
- Creates a unity3d directory in `XDG_CONFIG_HOME` and in `~/.config`, aswell as in `XDG_DATA_HOME` and `~/.local/share` which is weird and clogs up my `~/`.
- Fails to build to webGL.
- has problems with window size which makes it slow when hovering over buttons. 
- Hard to make it work with `neovim`.

I can probably fix some of these but seriously wtf is this.

**Comment by Aiz0** [comment]
I created a wrapper script for unityhub and unity editor where `HOME` is set to another directory and XDG envars are reset.
Doing this solved some problems I've had with unity. I am going to test it, clean up `~/` and add the wrapper scripts to this repository.

**Comment by Aiz0** [comment]
I've added the `unityhub` wrapper script. I've done a small amount of testing but I still need to test if it works with `neovim` now and if building to webGL works.

I did not upload a unity editor wrapper-script since I've been able to add and open projects with `unityhub` when I switched to using the wrapper-script.

I discovered some other bugs that I've been able to solve by using the wrapper-script.

- Packages are broken in releases 2020 and beyond
- Unable to download old releases via unity assistant on unity LTS releases page.

By switching to the wrapper-script both these bugs were solved.
I need to do further testing while working on my unity-project
